src/wazuh_db/wdb.c

```diff
@@ -1049,6 +1049,52 @@ cJSON* wdb_exec_stmt_sized(sqlite3_stmt * stmt, const size_t max_size, int* stat
     return result;
 }
 
+int wdb_exec_stmt_send(sqlite3_stmt* stmt, int peer) {
+    if (!stmt) {
+        mdebug1("Invalid SQL statement.");
```

Comment:
This error message talks about an "invalid" statement, but in fact it checks for NULL. Could it be rephrased ?

Reply:
Based on other exec_stmt methods that log the same message, I think they refer to the nullity of the statement as an "invalid" statement because these methods are called after preparing the statement, and a NULL statement is the return when the statement failed to be prepared. So, a NULL statement should represent an invalid statement as it was unable to prepare.

Reply:
Ok !